Sasquatch/Config/Sasquatch.xcconfig

```diff
@@ -1,5 +1,12 @@
 #include "../../Config/App.xcconfig"
 
+ARCHS[sdk=iphoneos*] = armv7 armv7s arm64 arm64e
+ARCHS[sdk=watchos*] = armv7k arm64_32
+ARCHS[sdk=iphonesimulator*] = x86_64 i386
```

Comment:
We have `MS_SIM_ARCHS` defined in `Global.xcconfig`.

Reply:
the apps are currently NOT using `Global.xcconfig` and I'd rather not introduce this change in this PR.